src/configure/configurers/azurePipelineConfigurer.ts

```diff
@@ -127,6 +159,62 @@ export class AzurePipelineConfigurer implements Configurer {
         return path.join(inputs.sourceRepository.localPath, await LocalGitRepoHelper.GetAvailableFileName('azure-pipelines.yml', inputs.sourceRepository.localPath));
     }
 
+    public async checkInPipelineFileToRepository(inputs: WizardInputs, localGitRepoHelper: LocalGitRepoHelper): Promise<string> {
+
+        let commitMessage: string;
+        let initializeGitRepository = !inputs.sourceRepository.remoteUrl;
+
+        if (!inputs.sourceRepository.remoteUrl) {
+            let repositoryName = path.basename(inputs.sourceRepository.localPath).trim().replace(/[^a-zA-Z0-9-]/g, '');
+            let repository = await this.azureDevOpsClient.createRepository(inputs.organizationName, inputs.project.id, repositoryName);
+
+            inputs.sourceRepository.repositoryName = repository.name;
+            inputs.sourceRepository.repositoryId = repository.id;
+            inputs.sourceRepository.remoteUrl = repository.remoteUrl;
+            commitMessage = 'We will intialize your workspace as git repository and commit all changes';
```

Comment:
We can move the text to messages.ts or somewhere else

Reply:
Moved it, this was a placeholder for actual message. @pulkitaggarwl  Can you suggest the commit message to be shown in case for non git repository.